_health-care/_js/components/personal-information/NameAndGeneralInfoSection.jsx

```diff
@@ -37,9 +37,7 @@ class NameAndGeneralInfoSection extends React.Component {
               <h4>Date of Birth</h4>
               <span className="usa-form-hint usa-datefield-hint" id="dobHint">For example: 04 28 1986</span>
               <DateInput
-                  day={this.props.data.dateOfBirth.day}
-                  month={this.props.data.dateOfBirth.month}
-                  year={this.props.data.dateOfBirth.year}
+                  value={this.props.data.dateOfBirth}
```

Comment:
Gotcha.

I would prefer to revert these for now.

Overall, it feels more awkward to have an object come in than individual parameters from an API perspective. It's like doing

`function renderDate(day, month, year)` vs `function renderDate(jsonOfDate)`.

Same argument could be made for some of other other more complex components (eg, FullName).